cmd/ci-op-configs-mirror/main.go

```diff
@@ -74,12 +75,23 @@ func main() {
 			rbc.CanonicalGoRepository = strP(fmt.Sprintf("github.com/%s/%s", repoInfo.Org, repoInfo.Repo))
 		}
 
-		repoInfo.Org = o.toOrg
+		if rbc.ReleaseTagConfiguration != nil {
+			filterReleaseTagConfiguration(rbc.ReleaseTagConfiguration)
+		}
+
+		if rbc.BuildRootImage.ImageStreamTagReference != nil {
```

Comment:
Can we dereference a nil here if `BuildRootImage` is nil?

Reply:
addressed